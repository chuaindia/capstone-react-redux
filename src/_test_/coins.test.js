import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { MemoryRouter as Router } from 'react-router-dom';
import Coinsdisplay from '../components/coins';
import '@testing-library/jest-dom';
import store from '../redux/configureStore';

it('renders correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Router>
        <Coinsdisplay />
      </Router>
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
