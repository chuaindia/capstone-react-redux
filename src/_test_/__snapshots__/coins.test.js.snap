// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<tr>
  <td>
    <a
      href="/detail/:undefined"
      onClick={[Function]}
    >
      <p
        className="arrow-container"
      >
         
        <i
          className="arrow right"
          label="but"
          onClick={[Function]}
          role="button"
          tabIndex={0}
        />
      </p>
    </a>
  </td>
  <td />
  <td
    className="price"
  />
</tr>
`;
